src/drivers/imu/mpu9250/mpu9250.cpp

```diff
@@ -995,11 +1047,26 @@ MPU9250::gyro_ioctl(struct file *filp, int cmd, unsigned long arg)
 	}
 }
 
+int
+MPU9250::select_register_bank(uint8_t bank) {
+	if (bank != _selected_bank ) {
```

Comment:
Shouldn't we set `_selected_bank` in here?

Reply:
Thanks for another catch!